jupyterlab_gcpscheduler/src/components/dialog.tsx

```diff
@@ -169,36 +174,57 @@ export class SchedulerDialog extends React.Component<Props, State> {
     }
   }
 
+  handleClick(event: React.MouseEvent<HTMLButtonElement>) {
+    this.setState({ anchorEl: event.currentTarget });
+  }
+
+  handleClose() {
+    this.setState({ anchorEl: null });
+  }
+
   render() {
     const projectId = this.getProjectId();
     return (
-      <Dialog open={this._isOpen()} scroll="body">
+      <Dialog open={this._isOpen()}>
         {this.state.showCreateForm && (
           <header className={localStyles.header}>
             <span className={localStyles.title}>
               Create a notebook{' '}
               {this.state.creatingExecution ? 'execution' : 'schedule'}{' '}
               <Badge value="alpha" />
             </span>
-            <IconButtonMenu
-              menuItems={menuCloseHandler => [
-                <SmallMenuItem key="viewAllExecutions">
-                  <a
-                    href={`${SCHEDULER_LINK}?project=${projectId}`}
-                    target="_blank"
-                    onClick={menuCloseHandler}
-                  >
-                    View all scheduled executions
-                  </a>
-                </SmallMenuItem>,
-                <SmallMenuItem
-                  key="reset"
-                  onClick={() => this._onResetSettings(menuCloseHandler)}
+            <IconButton onClick={this.handleClick}>
```

Comment:
Was there an issue with modifying the items passed to the shared `IconButtonMenu` component?

Reply:
Ahhh, no, I just wanted the styles to be consistent with the sidebar Menu as well.

Reply:
Ahh I see. It looks like there's a lot of boilerplate added in both places for some of the click handling? Would it be possible to factor the new styles out to the shared `IconButtonMenu` component? It looks like Scheduler is the only extension to use it anyways.

Reply:
Ooo okay. Can do that!

Reply:
Done!